src/ops/info.rs

```diff
@@ -328,3 +336,24 @@ fn validate_locked_and_frozen_options(
     }
     Ok(())
 }
+
+fn try_get_msrv_from_nearest_manifest_or_ws(
```

Comment:
This feels a bit backwards to me.

I feel like we should do the look up of workspace/package like other cargo commands, pretending `--manifest-path` is `None` (since we don't have that argument).

Reply:
I am not sure if I understand you correctly. Did you mean we should not expose the manifest as a parameter because we can not specify it? Because we always use `config.cwd()` here.

I've changed it to use config as a parameter instead of calling `config.cmd()` from the top level.

Reply:
Currently, we
1. Find the workspace
2. Find the parent package and ensure its a part of the above workspace

Normally, cargo commands instead
1. Find the parent package
2. Load the workspace for the above parent package

Being like other cargo commands would
- Be more consistent, making it easier to jump between commands and reduce the chance of bugs due to unawareness of corner cases like this
- Avoid duplicating the manifest discovery code
- Avoid having to verify the package/workspace are in sync
- Reduce risk for bugs in the avoided code

Reply:
Got it. Totally agree. Thanks! Updated.